cmd/fyne/internal/commands/build.go

```diff
@@ -60,6 +66,24 @@ func (b *builder) build() error {
 	return err
 }
 
+func (b *builder) generateMetaLDFlags() string {
+	var vars []string
```

Comment:
Wouldn't it be possible to instead just have an array here?

Reply:
I’m sorry I don’t really understand. Do you mean it should be fixed length?

Reply:
Sorry, I think I miss targeted the wrong line. I was talking about that series of if below which repeat the same code pretty much all over the place. Would be nicer if it was just an array being processed. Would be easier and cleaner also to extend over time.

Reply:
The alternative seems to be this, is it better?
```go
	buildIDString := ""
	if b.buildNum > 0 {
		buildIDString = strconv.Itoa(b.buildNum)
	}
	inserts := [][2]string{
		{"MetaID", b.id},
		{"MetaName", b.name},
		{"MetaVersion", b.version},
		{"MetaID", buildIDString},
	}
	var vars []string
	for _, item := range inserts {
		if item[1] != "" {
			vars = append(vars, "-X 'fyne.io/fyne/v2/internal/app."+item[0]+"="+item[1]+"'")
		}
	}
```

Reply:
Seems better and easier to extend.

Reply:
Cool, done